### PR TITLE
Fix bugs from migration

### PR DESCRIPTION
**Pull request by k4ey**
All major migration bugs resolved.

### CODE REVIEW

File: common/src/main/java/com/theincgi/advancedmacros/hud/hud2D/Hud2d_itemIcon.java
Comment (TheIncgi) [inline common/src/main/java/com/theincgi/advancedmacros/hud/hud2D/Hud2d_itemIcon.java:1]: Formatting on this one seems odd, github shows `^M` 
might just be line endings

Comment (TheIncgi) [reply common/src/main/java/com/theincgi/advancedmacros/hud/hud2D/Hud2d_itemIcon.java:1]: can merge it like that for now, can fix later

File: common/src/main/resources/assets/advancedmacros/scripts/changelogviewer.lua [hunk 1]
```diff
@@ -3,7 +3,6 @@ local args = {...}
 local settings = getSettings()
 settings.changeLog = settings.changeLog or {}
 local cl = settings.changeLog
-cl.showOnStartup = cl.showOnStartup==nil or cl.showOnStartup
```
Comment (TheIncgi) [inline common/src/main/resources/assets/advancedmacros/scripts/changelogviewer.lua:6]: Removed showing change log by default?

Also, would be good to get a list of key changes if anything has broke/been patched/added.
Are script GUIs working currently?

File: common/src/main/java/com/theincgi/advancedmacros/AdvancedMacros.java [hunk 2]
```diff
@@ -0,0 +1,491 @@
...
+import java.util.jar.JarFile;
+
+public class AdvancedMacros {
+
+    /**
+     * advancedMacros
+     */
+    public static final String MOD_ID = "advancedmacros";
+
+    public static final String VERSION = "11";
```
Comment (TheIncgi) [inline common/src/main/java/com/theincgi/advancedmacros/AdvancedMacros.java:76]: `11.0.0`?

File: fabric/src/main/resources/fabric.mod.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
+{
+  "schemaVersion": 1,
+  "id": "advancedmacros",
+  "version": "${version}",
+  "name": "Advanced Macros",
+  "description": "The Advanced Macros mod lets you write Lua scripts to automate just about anything in in Minecraft.",
+  "authors": [
+    "TheIncgi"
```
Comment (TheIncgi) [inline fabric/src/main/resources/fabric.mod.json:8]: Can add contributors if they want to be included :)

**Review by TheIncgi — APPROVED** [review]
All minor comments.
Also, not sure currently how many other things still need fixing, but happy to get it into the repo :) 

Great work from everyone involved 🎉

File: common/src/main/resources/assets/advancedmacros/scripts/changelogviewer.lua (hunk 1 above)
Comment (TheIncgi) [reply common/src/main/resources/assets/advancedmacros/scripts/changelogviewer.lua:6]: resolved in #47 's branch

File: fabric/src/main/resources/fabric.mod.json (hunk 3 above)
Comment (k4ey) [reply fabric/src/main/resources/fabric.mod.json:8]: sure!

**Pull request merged**